src/components/popup/Route.tsx

```diff
@@ -24,17 +24,21 @@ const Route: typeof BaseRoute = ({ path, component, children }) => {
   );
 };
 
-export const Wrapper = styled(motion.div)<{ responsive?: boolean }>`
+export const Wrapper = styled(motion.div)<{
+  responsive?: boolean;
+  expanded?: boolean;
+}>`
   position: relative;
   width: ${(props) => (props.responsive ? "100%" : "377.5px")};
-  min-height: 600px;
+  min-height: ${(props) => (props.expanded ? "100vh" : "600px")};
+  max-height: max-content;
+  background-color: rgb(${(props) => props.theme.background});
 `;
 
 const PageWrapper = styled(Wrapper)`
   position: absolute;
   top: 0;
   width: 100%;
-  background-color: rgb(${(props) => props.theme.background});
```

Comment:
Because in `PageWrapper` when filling bg in Receive/Send pages, it only filled up to the last element in them, not filling the whole height up to the navbar.

Try it out moving it back to page wrapper and you'll see the body bg in fullscreen